Automattic/wp-calypso
# Social previews: remove lodash and moment

## Proposed Changes

This PR removes `lodash` from the `social-previews` package.

## Testing Instructions



* Make sure you have a self-hosted Jetpack site
* Spin up Calypso by using a live link below or by running this branch locally
* Open the dev console
* VIsit `/marketing/traffic/:site`
* Click _Show Previewd_ in the _Website Meta_ section, then select any preview
* Check there's no error in the console

## Pre-merge Checklist
